cypress/integration/createEventWithoutStage.cy.js

```diff
@@ -0,0 +1,27 @@
+import { DIMENSION_ID_EVENT_DATE } from '../../src/modules/dimensionConstants.js'
+import { AEFI_PROGRAM, TEST_REL_PE_LAST_12_MONTHS } from '../data/index.js'
+import { selectEventProgram } from '../helpers/dimensions.js'
+import { clickMenubarUpdateButton } from '../helpers/menubar.js'
+import { selectRelativePeriod } from '../helpers/period.js'
+import { expectTableToBeVisible } from '../helpers/table.js'
+import { EXTENDED_TIMEOUT } from '../support/util.js'
+
+const program = AEFI_PROGRAM
```

Comment:
Unless you think the `AEFI_PROGRAM` will be useful for other tests I wouldn't create a const for it, here's an example of a single-use of the `MAL-CS` program where I just wrote it in plain text to keep things simple https://github.com/dhis2/line-listing-app/blob/b75b27b911b93ad189d1a041c64e8822d3a860e5/cypress/integration/repeatedEvents.cy.js#L187